<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NodeDiscoverTestClass</class>
 <widget class="QMainWindow" name="NodeDiscoverTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NodeDiscoverTest</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1" rowspan="2">
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="cbCycle">
           <property name="text">
            <string>循环发送</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>步长</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtStep">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>5000</string>
             </property>
             <property name="maxLength">
              <number>32767</number>
             </property>
             <property name="placeholderText">
              <string>ms</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>发送数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="tbSend"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>接收数据</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextBrowser" name="tbRecv"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QPushButton" name="btnSelectFile">
         <property name="text">
          <string>选择文件</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="edtPath"/>
       </item>
       <item row="5" column="0">
        <widget class="QPushButton" name="btnSendFile">
         <property name="text">
          <string>发送文件</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>255</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0" rowspan="2" colspan="5">
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="NodeDiscoverTest.qrc"/>
 </resources>
 <connections/>
</ui>
